modules/tinymce/src/core/main/ts/dom/NodeType.ts

```diff
@@ -14,6 +14,13 @@ const isRestrictedNode = (node: NullableNode): boolean => !!node && !Object.getP
 
 const isElement = isNodeType<HTMLElement>(1);
 
+const matchNodeName = <T extends Node>(name: string): (node: NullableNode) => node is T => {
+  const lowercasedName = name.toLowerCase();
```

Comment:
Maybe in the camel case better` lowerCasedName`

Reply:
I guess that makes sense that the toLowerCase function has a big C. So I changed both of these variables that have a lowercase C.